frontend/src/pages/Login/Login.jsx

```diff
@@ -1,17 +1,25 @@
-import { useState } from "react";
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
 import { useNavigate } from "react-router-dom";
 import * as s from "./Login.styles";
```

Comment:
```suggestion
import * as S from "./Login.styles";
```
사소하지만 첫글자이므로 대문자 S로 바꿔주면 더 보기 좋을 것 같습니다ㅎㅎ

Reply:
넵 수정해서 다음 커밋에 반영하겠습니당!!